src/App.js

```diff
@@ -1,13 +1,112 @@
 import React from 'react';
+import "./components/TodoComponents/Todo.css"
+import Todo from './components/TodoComponents/Todo';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      newTask: "",
+      todos: [
+        {
+          //id: new Date(),
+          id: 1,
+          task: "Go to a meeting in Maryland",
+          completed: false
+        },
+        {
+          //id: new Date(),
+          id: 2,
+          task: "Take the kids to Jurasic park",
+          completed: false
+          
+        },
+        {
+          //id: new Date(),
+          id: 3,
+          task: "Attend Organization end of the year party",
+          completed: false
+        },
+        {
+          //id: new Date(),
+          id: 4,
+          task: "Say no to offer from Wasteland",
+          completed: false
+        },
+        {
+          //id: new Date(),
+          id: 5,
+          task: "Go to the rendezvous for some downtime",
+          completed: false
+        },
+        {
+          //id: new Date(),
+          id: 6,
+          task: "Solve the mystery surrounding the dancing men",
+          completed: false
+        }
+      ]
+    }
+    this.handleAddTodo = this.handleAddTodo.bind(this);
+    //this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  
+  
+
+  handleAddTodo = (event) =>{
```

Comment:
you need to make it a variable if you want it to works --> const handleTodo = event => {